Simplenote/Date+Simplenote.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+extension Date {
+    func increased(by days: Int) -> Date? {
```

Comment:
```suggestion
    func increased(byDays days: Int) -> Date? {
```

Reply:
Otherwise when you invoke `date.increased(by: 123)` it's not clear what's going on